webapp/channels/src/components/drafts/draft_actions/schedule_post_actions/scheduled_post_actions.tsx

```diff
@@ -111,36 +119,38 @@ function ScheduledPostActions({scheduledPost, onReschedule, onDelete, channelDis
             />
 
             {
-                !scheduledPost.error_code && (
-                    <React.Fragment>
-                        <Action
-                            icon='icon-pencil-outline'
-                            id='edit'
-                            name='edit'
-                            tooltipText={editTooltipText}
-                            onClick={onEdit}
-
-                        />
-
-                        <Action
-                            icon='icon-clock-send-outline'
-                            id='reschedule'
-                            name='reschedule'
-                            tooltipText={rescheduleTooltipText}
-                            onClick={handleReschedulePost}
-                        />
-
-                        <Action
-                            icon='icon-send-outline'
-                            id='sendNow'
-                            name='sendNow'
-                            tooltipText={sendNowTooltipText}
-                            onClick={handleSend}
-                        />
-                    </React.Fragment>
-                )
+                showEditOption &&
+                <Action
+                    icon='icon-pencil-outline'
+                    id='edit'
+                    name='edit'
+                    tooltipText={editTooltipText}
+                    onClick={onEdit}
+
+                />
             }
 
+            {
+                showRescheduleOption &&
+                <Action
+                    icon='icon-clock-send-outline'
+                    id='reschedule'
+                    name='reschedule'
+                    tooltipText={rescheduleTooltipText}
+                    onClick={handleReschedulePost}
+                />
+            }
+
+            {
+                channelDisplayName && showSendNowOption &&
```

Comment:
Should `channelDisplayName` be included as part of `showSendNowOption`? I like naming those to explain what they do, so I was a bit surprised to see `channelDisplayName` here

Reply:
Done.